tests/examples/wibox/layout/defaults/manual.lua

```diff
@@ -0,0 +1,25 @@
+local generic_widget = ... --DOC_HIDE
+local wibox     = require("wibox") --DOC_HIDE
+
+local w1 = generic_widget() --DOC_HIDE
+w1.point  = {x=75,y=5} --DOC_HIDE
+w1.text   = "first" --DOC_HIDE
+w1.forced_width = 50 --DOC_HIDE
```

Comment:
Why the many doc hides? It might be confusing enough that `w1` and `w2` come out of nowhere while `third` is created via a call to `generic_widget()` (as the reader of the docs: Whatever that might be). But you also hide the `point` for `w1`.
Should perhaps more of this be done in a declarative way?